camera-manager.js

```diff
@@ -5,6 +5,8 @@ import metaversefile from 'metaversefile';
 import physicsManager from './physics-manager.js';
 import {shakeAnimationSpeed} from './constants.js';
 import Simplex from './simplex-noise.js';
+import { lerp } from 'three/src/math/MathUtils';
+import { localPlayer } from './players.js';
```

Comment:
Hmm, that works but I'm not sure if it's intended to be used like that.
You can get the localPlayer like this:
`const localPlayer = metaversefileApi.useLocalPlayer();`

Reply:
It's fine to import `localPlayer` like this if you are not an app.

Reply:
I just entirely removed the dependency on localplayer from within the aim functionality. I only need to pass in the players position once at the start of takeaim anyways